src/components/page-header/master-manager.tsx

```diff
@@ -0,0 +1,227 @@
+            core: p.core,
+            version: p.version,
+        };
+        param.components.forEach((c, i) => {
+            param.components[i].value = c.defaultValue;
+        });
+        downloadAs(`RMP_Master_Node_${new Date().valueOf()}.json`, 'application/json', JSON.stringify(param));
+    };
+
+    const handleRemove = (type: string) => {
+        graph.current
+            .filterNodes(
+                node =>
+                    graph.current.getNodeAttribute(node, 'type') === MiscNodeType.Master &&
+                    graph.current.getNodeAttributes(node)[MiscNodeType.Master]!.randomId === type
+            )
+            .forEach(node => {
+                graph.current.dropNode(node);
+            });
+        dispatch(refreshNodesThunk());
+        dispatch(saveGraph(graph.current.export()));
+    };
+
+    const handleSetLabel = (attr: MasterParam, label: string) => {
+        graph.current
+            .filterNodes(
+                node =>
+                    graph.current.getNodeAttribute(node, 'type') === MiscNodeType.Master &&
+                    graph.current.getNodeAttributes(node)[MiscNodeType.Master]!.randomId === attr.randomId
+            )
+            .forEach(node => {
+                graph.current.mergeNodeAttributes(node, { [MiscNodeType.Master]: { ...attr, label } });
+            });
+        dispatch(refreshNodesThunk());
+        dispatch(saveGraph(graph.current.export()));
+    };
+
+    const fields: RmgFieldsField[][] = list.map(attrs => {
+        return [
+            {
+                label: t('panel.details.masterManage.id'),
+                type: 'custom',
+                component: (
+                    <RmgLineBadge
+                        name={attrs.randomId}
+                        fg={MonoColour.white}
+                        bg={attrs.randomId === 'undefined' ? '#000000' : '#19B3EA'}
+                    />
+                ),
+            },
+            {
+                label: t('panel.details.masterManage.label'),
+                type: 'input',
+                value: attrs.label ?? attrs.randomId,
+                onChange: value => handleSetLabel(attrs, value),
+                hidden: attrs.randomId === 'undefined',
+            },
+            {
+                label: t('panel.details.masterManage.label'),
+                type: 'output',
+                value: attrs.label ?? attrs.randomId,
+                hidden: attrs.randomId !== 'undefined',
+            },
+            {
+                label: t('panel.details.masterManage.type'),
+                type: 'custom',
+                component: (
+                    <RmgLineBadge
+                        name={attrs.nodeType}
+                        fg={MonoColour.white}
+                        bg={attrs.nodeType === 'MiscNode' ? '#FF8651' : '#51BC00'}
+                    />
+                ),
+            },
+            {
+                label: '',
+                type: 'custom',
```

Comment:
This Flex could be just added after RmgFields in ModalBody since it is a custom components and uses nothing from rmg-components.